Localize/lang/strings.json

```diff
@@ -1,2 +1,3532 @@
 {
```

Comment:
Do you know why it looks like this PR added over 3,000 lines to `strings.json`? Maybe @rllyy97 knows?

Reply:
Yeah we're having a weird issue where the commit tasks keep adding/removing that file, the file should be there so this is fine